src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import { useState } from 'react';
 import './App.scss';
+// import classNames from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, TodoUser, User } from './interfaces';
+
+const todosUsersFiltered: TodoUser[] = todosFromServer.map((todo: Todo) => {
+  const matchUser = usersFromServer.find(
+    (user: User) => user.id === todo.userId,
+  );
+
+  return {
+    ...todo,
+    user: matchUser,
+  };
+});
 
 export const App = () => {
+  const [count, setCount] = useState(16);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [errTitle, setErrTitle] = useState(false);
+  const [errUser, setErrUser] = useState(false);
+  const [todoUsers, setTodoUser] = useState<TodoUser[]>(todosUsersFiltered);
+  const usersArr = [
+    {
+      id: 0,
+      name: 'Choose a user',
+    },
+    ...usersFromServer,
+  ];
+
+  const handleUserSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const userId = Number(event.target.value);
+
+    setSelectedUser(userId);
+    setErrUser(false);
+  };
+
+  const handlTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setErrTitle(false);
+  };
+
+  const submitButton = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    let hasError = false;
+
+    if (title === '') {
+      setErrTitle(true);
+      hasError = true;
+    } else {
+      setErrTitle(false);
+    }
+
+    if (selectedUser === 0) {
+      setErrUser(true);
+      hasError = true;
+    } else {
+      setErrUser(false);
+    }
+
+    if (hasError) {
+      return;
+    }
```

Comment:
can we simplify it?
let's try not to use else & let in code
 if (selectedUser === 0 || title === '') {
      setErrUser(true);
return
}